### ISSUE TITLE
[Skeleton Viewer] not working on MacOS 10.15 Catalina

### ISSUE THREAD

**Issue by yanjifa** [EsotericSoftware/spine-runtimes#1423]
Skeleton Viewer version is v3.7.94

When I  want open a new file，but the "open" button does not respond, some logs like this.

```
$ java -jar skeletonViewer.jar
2019-07-30 00:39:32.325 java[17450:329386] NSSoftLinking - The function 'SLSIsSuppressedByScreenTime' can't be found in the (null) framework.
2019-07-30 00:39:41.241 java[17450:329514] +[NSXPCSharedListener endpointForReply:withListenerName:]: an error occurred while attempting to obtain endpoint for listener 'com.apple.view-bridge': Connection interrupted
```

some good news, I also can directly specify the skeleton .json or .skel to open:

```
java -jar skeletonViewer.jar path\to\skeleton.json
```

**Comment by NathanSweet** [comment]
Catalina is in beta, we'd like to wait until it is a bit more mature before trying it on our machines.

Note this also happens with Spine's file dialogs. In Spine you can try holding `alt`/`meta` when opening the file dialog. This won't help in Skeleton Viewer I'm afraid.

**Comment by NathanSweet** [comment]
Has been reported as fixed by macOS Catalina 10.15 Beta (19A546d) from August 28, 2019. Can someone please confirm?
